core/src/main/scala/kafka/server/ReplicaFetcherThread.scala

```diff
@@ -72,7 +72,7 @@ class ReplicaFetcherThread(name: String,
       metrics,
       time,
       "replica-fetcher",
-      Map("broker-id" -> sourceBroker.id.toString).asJava,
+      Map("thread-id" -> name).asJava,
```

Comment:
@junrao Not sure I follow, since thread-name contains the broker-id and the thread-id (within the threads for that broker), is this logically the same as passing two tags, broker-id and thread-id?